webapp/public/locales/fr/translation.json

```diff
@@ -357,6 +357,9 @@
     "Move Section Down": "Faire Descendre Section",
     "Move Section Up": "Faire Monter Section",
     "No - Don't delete": "Non - Ne pas Suprimer",
-    "Yes - Delete": "Oui - Suprimer"
+    "Yes - Delete": "Oui - Suprimer",
+    "Review Details": "Review Details",
+    "Total": "Total",
+    "Candidate": "Candidate"
```

Comment:
We will need appropriate French translations for these new keys.

Reply:
Please use Google translate to get these for now

Reply:
Done in https://github.com/deep-learning-indaba/Baobab/pull/871/commits/5fbf0b8f6b6adebc8ccad9df1073a30bfd3052f4